setup.py

```diff
@@ -33,6 +33,7 @@
     'Programming Language :: Python :: 3.8',
```

Comment:
Should we remove 3.8 you think?

Reply:
I think we can wait until the inference server drops support maybe?